ambry-utils/src/main/java/com.github.ambry.utils/Utils.java

```diff
@@ -748,6 +749,24 @@ public static long addSecondsToEpochTime(long epochTimeInMs, long deltaTimeInSec
     return data;
   }
 
+  /**
+   * Read the given number of bytes from the given {@link InputStream} into the given {@link OutputStream}
+   * @param inStream the {@link InputStream} from which to read.
+   * @param outStream the {@link OutputStream} to which to write.
+   * @param size the number of bytes to be transferred.
+   * @throws IOException if there is an exception reading or writing to the given streams.
+   */
+  public static void transferBytes(InputStream inStream, OutputStream outStream, long size) throws IOException {
```

Comment:
function _may_ no longer be needed

Reply:
if you want to retain this anyway, a test is needed

Reply:
This is potentially useful going forward, but will remove for now, don't want to bloat this patch further.